src/python/pants/base/workunit.py

```diff
@@ -246,6 +246,18 @@ def root(self):
       ret = ret.parent
     return ret
 
+  def is_background(self, background_root_workunit):
+    """
+    :API: public
+    """
+    ret = self
```

Comment:
nit: let's rename `ret` (to `curr_workunit`?), since it no longer carries the meaning of the possible return value, unlike the methods above.